server/models/ChartsModel/index.js

```diff
@@ -10,7 +10,9 @@ const $updatedAt = '$updatedAt'
 
 const ChartsModel = {
   create: async (db, chartAttributes) =>
-    await db.collection(collections.charts).insertOne(chartAttributes),
+    await db
+      .collection(collections.charts)
+      .insertOne({ ...chartAttributes, updatedAt: new Date() }),
```

Comment:
```suggestion
    const now = new Date()

    await db
      .collection(collections.charts)
      .insertOne({ ...chartAttributes, createdAt: now, updatedAt: now }),
```

Reply:
i'm confused, the prior comments made it seem that we just need the updatedAt property..and now we are adding a createdAt property during document creation, which makes sense but now we are jamming an updated property at creation... it makes sense to have the updatedAt property when updating and creating when created

Reply:
we want both timestamp properties on every record. When an object is created, the created and updated timestamps should be equal. When it is updated, the updated timestamp should be changed to the time of the update.

Reply:
Okay I like this description I'll add it to the ticket